lib/pf/radius.pm

```diff
@@ -938,10 +941,20 @@ Handle NTLM caching if necessary
 
 sub handleNtlmCaching {
     my ($self, $radius_request) = @_;
+    my $logger = get_logger;
     my $domain = $radius_request->{"PacketFence-Domain"};
+    my $usedNtHash = $radius_request->{"PacketFence-NTCacheHash"};
+
     if($domain && isenabled($ConfigDomain{$domain}{ntlm_cache}) && isenabled($ConfigDomain{$domain}{ntlm_cache_on_connection})) {
-        my $client = pf::api::queue->new(queue => "general");
-        $client->notify("cache_user_ntlm", $domain, $radius_request->{"Stripped-User-Name"});
+        my $CHI_CACHE = pf::CHI->new( namespace => 'ntlm_cache_username_lookup' );
+        my $cache_key = "$domain.$radius_request->{'Stripped-User-Name'}";
+        my $username = $CHI_CACHE->get($cache_key);
+        if (defined($usedNtHash) && $usedNtHash && defined($username)) {
+            update_user_in_redis_cache($domain,$username);
```

Comment:
To keep things consistent this should be called on $self like the other subs of this module